tasks.py

```diff
@@ -948,8 +963,8 @@ def ci_tests(  # noqa: PLR0913
         "-rEf",
     ]
 
-    if xdist:
-        pytest_cmds.append("-n auto")
+    # if xdist:
+    #     pytest_cmds.append("-n auto")
```

Comment:
Also noticed this got commented out.
If this is causing problems I would suggest we instead change the default of `xdist = False` instead of removing this completely